### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by henry-dura**
In this PR, I did the following:

- Set it up with `webpack`
- Created an `index.js`  file and set an array of some simple to-do tasks (array of objects).
- iterate d over the tasks array and populate the `HTML` list item element dynamically.
- Created a `style.css ` which is loaded by `Webpack Style/CSS Loader`. 
- All my source files (`index.html`, `index.js` and `style.css`) are located in `/src` directory and my `distribution files ` will be generated by `webpack` and served by `webpack dev server` from `/dist ` folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+
+
+### Install
+
+Install this project with:
+```sh
+    cd your-desired-folder
+    git clone https://github.com/henry-dura/To-Do-List.git
+```
+
```
Comment (t-yanick) [inline README.md:61]: - [ ] Kindly add the procedure to run/start your project with webpack as this will make your documentation more professional

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @henry-dura , 

### Good job so far!

### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Linters are all passing
:heavy_check_mark: Excellent use of Webpack

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -50,12 +53,17 @@ Install this project with:
 ```sh
     cd your-desired-folder
     git clone https://github.com/henry-dura/To-Do-List.git
+    - Open it in the Code editor (Visual studio code, sublime text e.t.c) installed on your computer.
+    - In the terminal, run npm start to start the project
```
Comment (HAMMAS-SALEEM) [inline README.md:57]: **[Optional]**

- [ ] It's great that you have added the **npm start** command :blush: but I guess you need to install dependencies using the **npm install** command :thinking: so it should come before **npm start** command :+1:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @henry-dura 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**